lightning/src/ln/channel.rs

```diff
@@ -1983,7 +1977,12 @@ impl<Signer: Sign> Channel<Signer> {
 		self.counterparty_selected_channel_reserve_satoshis = Some(msg.channel_reserve_satoshis);
 		self.counterparty_htlc_minimum_msat = msg.htlc_minimum_msat;
 		self.counterparty_max_accepted_htlcs = msg.max_accepted_htlcs;
-		self.minimum_depth = Some(msg.minimum_depth);
+
+		if peer_limits.trust_own_funding_0conf {
+			self.minimum_depth = Some(msg.minimum_depth);
+		} else {
+			self.minimum_depth = Some(cmp::max(1, msg.minimum_depth));
```

Comment:
Per the spec:
```
  - if `channel_type` includes `option_zeroconf`:
    - MUST set `minimum_depth` to zero.
```

Wondering if this is non-compliant with that?

Reply:
I dont believe we ever set the 0conf channel type at all, no?

Reply:
Hmm, this is under the requirements for `accept_channel`. IIUC, if we receive an `open_channel` with `option_zeroconf`, we MUST set min depth to 0 in `accept_channel`?